src/sql/src/session/vars.rs

```diff
@@ -1376,6 +1376,15 @@ pub const ENABLE_CONSOLIDATE_AFTER_UNION_NEGATE: ServerVar<bool> = ServerVar {
     internal: false,
 };
 
+/// Whether compute rendering should type-specialize arrangements.
```

Comment:
Can you add a note here describing why it's important that we don't define this using the macro?

Reply:
With https://github.com/MaterializeInc/materialize/pull/22374, it's now clarified that it is safe to use `feature_flags!` here, so I changed the code back to doing so.